app/app.go

```diff
@@ -181,7 +183,14 @@ func NewWasmApp(
 	// better way to get this dir???
 	homeDir := viper.GetString(cli.HomeFlag)
 	wasmDir := filepath.Join(homeDir, "wasm")
-	app.wasmKeeper = wasm.NewKeeper(app.cdc, keys[wasm.StoreKey], app.accountKeeper, app.bankKeeper, wasmRouter, wasmDir)
+
+	// Read LRU cache size from config (app.toml), defaults to `0`
+	cacheSize := uint64(viper.GetInt64(flagLRUCacheSize))
+	// Read query gas limit from config
+	smartQueryGasLimit := uint64(viper.GetInt64(flagQueryGasLimit))
+
+	app.wasmKeeper = wasm.NewKeeper(app.cdc, keys[wasm.StoreKey], app.accountKeeper, app.bankKeeper, wasmRouter,
```

Comment:
Even if we don't need to modify the sdk Config, how about putting these into a `WasmConfig` struct, so we don't need to add more args here, even if we add more options to the config (and I bet we will)

Reply:
Makes sense. Updated the required changes and its r4r